### PR TITLE
UI fixes

### LINKED ISSUE #296: Problem with the 3 dots in send modal, responsive design
### Expected behaviour
Aligned with the form.

### Actual behaviour
Below the form.

### Steps to reproduce
Open send modal. Increase the size of the window to the biggest size.

unaligned-3dots

### PR DESCRIPTION
**Pull request by alepop**
Resolve #296, #297, #298, #302

### CODE REVIEW

File: src/components/send/send.pug [hunk 1]
```diff
@@ -12,16 +12,16 @@ div.dialog-send(aria-label='Send funds')
             div(ng-messages='$ctrl.transferForm.recipient.$error')
               div(ng-message='required') Required
               div(ng-message='pattern') Invalid
-          div(layout="row")
-            md-input-container.md-block.flex-95
+          div(style={'position': 'relative'})
```
Comment (alihaghighatkhah) [inline src/components/send/send.pug]: Please don't use inline styling.

**Review by reyraa — CHANGES_REQUESTED** [review]
Except the minor change I've requested, all good! Thank you

**Review by slaweet — APPROVED** [review]
All looks good now.

**Review by reyraa — APPROVED** [review]

**Comment by slaweet** [comment]
@alepop next time you Resolve multiple issues in one PR, it needs to specify with each explicitly:
https://help.github.com/articles/closing-issues-via-commit-messages/#closing-multiple-issues

**Comment by alepop** [comment]
@slaweet ok. I don't know about this feature.

**Pull request merged**